ed_sensor_integration/tools/fitter_viz.cpp

```diff
@@ -0,0 +1,425 @@
+#include <ros/init.h>
+#include <ros/node_handle.h>
+
+#include <ed/models/model_loader.h>
+#include <ed/world_model.h>
+#include <ed/update_request.h>
+#include <ed/serialization/serialization.h>
+#include <ed/io/json_reader.h>
+#include <ed/kinect/updater.h>
+#include <ed/kinect/fitter.h>
+
+#include <tue/filesystem/crawler.h>
+
+#include <rgbd/image.h>
+#include <rgbd/serialization.h>
+#include <rgbd/view.h>
+
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+
+#include <tue/config/read.h>
+#include <tue/config/reader.h>
+#include <tue/config/data_pointer.h>
+
+#include <ed/entity.h>
+
+#include "geolib/math_types.h"
+#include <tf2/convert.h>
+#include <tf2/utils.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+
+#include <fstream>
+#include <vector>
+
+// ----------------------------------------------------------------------------------------------------
+
+struct Snapshot
+{
+    rgbd::ImagePtr image;
```

Comment:
Any reason to use an `ImagePtr` instead of an `Image`?

Reply:
Not that I know of. this was copied from ed_segmenter